include/amqpcpp.h

```diff
@@ -8,6 +8,8 @@
 
 #pragma once
 
+#define NOMINMAX
```

Comment:
Took the easy way out here.
Windows.h normally defines 'min' and 'max' as macros, which prevents those symbols from being used as member functions, etc..  This #define prevents that behavior, but can cause compilation issues in any 3rd party libraries that depend on the legacy behavior (and is hard to track down unless one knows about it).